ftw/testbrowser/form.py

```diff
@@ -206,10 +206,8 @@ def submit(self, button=None):
                 return buttons.first.click()
 
         extra_values = None
-        if button and button.attrib.get('name', None) and \
-                button.attrib.get('value', None):
-            extra_values = {button.attrib['name']: button.attrib['value']}
-
+        if button and button.attrib.get('name', None):
+            extra_values = {button.attrib['name']: button.attrib.get('value', '')}
```

Comment:
If I understand corectly, this will send an empty string as value, e.g. `?thename=&...` for `<button name="thename">`.
Did you verify what the browser do, e.g. Chrome?